src/abstractdataframe/reshape.jl

```diff
@@ -170,7 +170,7 @@ function unstack(df::AbstractDataFrame, rowkey::Int, colkey::Int, value::Int)
     Nrow = length(refkeycol.pool)
     Ncol = length(keycol.pool)
     # TODO make fillNA(type, length)
-    payload = DataFrame(Any[DataArray(fill(valuecol[1], Nrow), fill(true, Nrow)) for i in 1:Ncol], map(@compat(Symbol), keycol.pool))
+    payload = DataFrame(Any[DataArray(eltype(valuecol), (Nrow)) for i in 1:Ncol], map(@compat(Symbol), keycol.pool))
```

Comment:
`(Nrow)` is strictly equivalent to `Nrow` (you'd have to write `(Nrow,)` if you wanted a tuple), so better use the simpler version.